core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -253,13 +257,15 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("deserializer for values")
       .ofType(classOf[String])
+    val enableLifecycleLoggingOpt = parser.accepts("enable-lifecycle-logging", "Log lifecycle events of the consumer.")
```

Comment:
My main question about this approach is that I'm not sure which lifecycle events are best to "log". If we're providing this flag, it seems surprising to only log a clean shutdown.

And, does "log" mean print?